src/Components/App/App.js

```diff
@@ -1,24 +1,12 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react'
 import './App.css';
+import Books from '../Books/Books'
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+     <h1>Hiyas</h1>
+     <Books />
```

Comment:
Now that I look at all of this, I wonder if there are ways we can make our component names more semantic! It's definitely going to get confusing to distinguish between `Book` and `Books`- I wonder if we should shift to something like `BookCard` and 'BookContainer`...

Reply:
I like this idea Jake, it could be easy to get Book and Books mixed up.